src/assets/icon/IconFollowDomain.js

```diff
@@ -9,7 +9,7 @@ function IconFollowDomain(props) {
         fill-rule="evenodd"
         clip-rule="evenodd"
         d="M10.1943 1.11509C10.1943 0.499245 9.6951 0 9.07925 0C8.4634 0 7.96415 0.499245 7.96415 1.11509L7.96415 7.80566L1.1151 7.80566C0.499245 7.80566 0 8.3049 0 8.92075C0 9.5366 0.499245 10.0358 1.11509 10.0358H7.96415L7.96415 16.8849C7.96415 17.5008 8.4634 18 9.07924 18C9.6951 18 10.1943 17.5008 10.1943 16.8849L10.1943 10.0358H16.8849C17.5008 10.0358 18 9.5366 18 8.92075C18 8.3049 17.5008 7.80566 16.8849 7.80566L10.1943 7.80566L10.1943 1.11509Z"
-        fill="#00ADB5"
+        fill={COLORS.anon_primary}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fill color for the SVG path has been changed from a hardcoded value to a reference from the `COLORS` object. This is a good practice as it allows for easier updates and consistency in color usage across the application.
```diff
- fill="#00ADB5"
+ fill={COLORS.anon_primary}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->